docs/recipes/unit-testing.md

```diff
@@ -81,3 +81,28 @@ describe('Zoo', () => {
   });
 });
 ```
+
+Usually in your application you have selectors created dynamically using the createSelector function:
+
+```TS
+export class ZooSelectors {
+  static animals = () => {
+    return createSelector([ZooState], (state: ZooStateModel) => {
+      return state.animals;
+    });
+  }
```

Comment:
I think that this example could be improved. You don't need to use the `createSelector` function when you are creating a selector that is not parameterised. You can just use the `@Selector` decorator. Your code here wuold better be written as:
```TS
@Selector([ZooState])
animals(state: ZooStateModel) {
  return state.animals;
}
```
So I think that a better example of a `createSelector` usage would be:
```TS
  static animalNames = (type: string) => {
    return createSelector([ZooState], (state: ZooStateModel) => {
      return state.animals
        .filter((animal) => animal.type === type )
        .map((animal => animal.name ));
    });
  }
```
The test would need to be adjusted accordingly